Automattic/wp-calypso
# Anchor: Add and update unit tests for hooks

#### Changes proposed in this Pull Request

* Update unit tests for `useIsAnchorFmParams` to account for the other query params in addition to episode ID
* Add unit tests for `useIsAnchorFm` hook

#### Testing instructions

* Switch to this PR
* Run `jest landing/stepper/hooks/test/use-is-anchor-fm.tsx` and `jest landing/stepper/hooks/test/use-is-anchor-fm-params.tsx`
* Tests should pass

Related to #63049
